build/configs/rtl8721csm/security_hal_test_tz/Make.defs

```diff
@@ -21,10 +21,11 @@ include ${TOPDIR}/tools/Config.mk
 ARCH_FAMILY = $(patsubst "%",%,$(CONFIG_ARCH_FAMILY))
 include ${TOPDIR}/arch/$(CONFIG_ARCH)/src/$(ARCH_FAMILY)/Toolchain.defs
 
-ifeq ($(CONFIG_AMEBAD_TRUSTZONE),y)
 LDSCRIPT = rlx8721d_img2_ns.ld
-else
-LDSCRIPT = rlx8721d_img2_is.ld
```

Comment:
Let's keep original code. Because this could be a base of other new config.

Reply:
Reverted